storybook/preview.js

```diff
@@ -8,7 +8,23 @@ import { addDecorator } from '@storybook/react';
 /**
  * Internal dependencies
  */
-import './wordpress/css/wp-admin.min.css';
+
+import './wordpress/css/common.min.css';
```

Comment:
Manually doing these imports now because we need to import them statically when generating the static storybook we deploy to gh pages.